includes/class-coblocks-labs.php

```diff
@@ -115,6 +129,13 @@ public function propagate_settings() {
 			update_option( 'coblocks_site_design_controls_enabled', 0 );
 			update_option( 'coblocks_layout_selector_controls_enabled', 0 );
 		}
+
+		// Site design is disabled when in GSE context.
+		$is_go_se = self::is_go_se();
```

Comment:
Should this variable and function be `is_go_fse`? Or is `se` correct?

Reply:
> Site Editing (formerly known as Full Site Editing or FSE)
https://wordpress.com/support/site-editing/

FSE is dead now and SE is the new norm. I have been pushing for GSE as a reference for that reason.